right/src/utils.c

```diff
@@ -103,3 +112,39 @@ void Utils_SafeStrCopy(char* target, const char* src, uint8_t max) {
     memcpy(target, src, stringlength);
     target[stringlength-1] = '\0';
 }
+
+const char* Utils_KeyAbbreviation(key_state_t* keyState)
+{
+    if (true) {
```

Comment:
What's the point of this condition if only this branch runs?

Reply:
Clearly to preserve the other branch in case it is needed later.

On second thought, I will remove the other branch.